src/main/java/de/tum/in/www1/artemis/web/rest/ResultResource.java

```diff
@@ -314,7 +313,7 @@ private boolean areManualResultsAllowed(final Exercise exerciseToBeChecked) {
             final var exercise = (ProgrammingExercise) exerciseToBeChecked;
             final var relevantDueDate = exercise.getBuildAndTestStudentSubmissionsAfterDueDate() != null ? exercise.getBuildAndTestStudentSubmissionsAfterDueDate()
                     : exercise.getDueDate();
-            return exercise.getAssessmentType() == AssessmentType.SEMI_AUTOMATIC && (relevantDueDate == null || !relevantDueDate.isBefore(ZonedDateTime.now()));
```

Comment:
This seemed like an error to me: The manual assessment is **not** allowed if the due date has passed (due date < now).